service/update.check/pom.xml

```diff
@@ -88,6 +81,8 @@
                 <configuration>
                     <additionalClasspathElements>
                         <additionalClasspathElement>${basedir}/src/test/resources/gateway.server-5.0.0.8.jar</additionalClasspathElement>
+                        <argLine>${surefireArgLine}</argLine>
+                        <skipTests>${skipTests}</skipTests>
                     </additionalClasspathElements>
                 </configuration>
             </plugin>
```

Comment:
This module has its own configuration of surefire because it needs the additionalClasspathElement for one of its tests